pronto_mia/pubspec.yaml

```diff
@@ -1,61 +1,45 @@
 name: pronto_mia
 description: A new Flutter application.
 
-# The following line prevents the package from being accidentally published to
-# pub.dev using `pub publish`. This is preferred for private packages.
-publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+publish_to: 'none'
 
-# The following defines the version and build number for your application.
-# A version number is three numbers separated by dots, like 1.2.43
-# followed by an optional build number separated by a +.
-# Both the version and the builder number may be overridden in flutter
-# build by specifying --build-name and --build-number, respectively.
-# In Android, build-name is used as versionName while build-number used as versionCode.
-# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
-# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
-# Read more about iOS versioning at
-# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 version: 0.0.2+1
 
 environment:
   sdk: ">=2.7.0 <3.0.0"
 
-# TODO: Fix version incompatibilities
 dependencies:
   date_time_picker: ^2.0.0
-  file_picker: ^3.0.1
-  firebase_core:
-  firebase_messaging:
+  file_picker: ^3.0.0
+  firebase_core: ^1.1.0
+  firebase_messaging: ^9.1.0
   flutter:
     sdk: flutter
-  flutter_cache_manager:
+  flutter_cache_manager: ^2.1.0
   flutter_pdfview: ^1.1.0
-  flutter_secure_storage:
-  flutter_svg:
+  flutter_secure_storage: ^4.0.0
+  flutter_svg: ^0.19.0 # ^0.22.0 depends on http ^0.13.0
   global_configuration: ^1.6.0

```

Comment:
So does changing this not improve anything?

Reply:
The problem with `global_configuration` was that it depends on http v0.12.0, while other libraries depend on v0.13.0 on newer versions. `graphql` is dependent on http v0.12.0 in **any** version, so we need to keep http on it anyway and therefore leave some of the dependencies that would need v0.13.0 for newer versions behind until graphql updates its dependencies.